opts/opts.go

```diff
@@ -277,7 +283,7 @@ func ValidateExtraHost(val string) (string, error) {
 }
 
 func ValidateLabel(val string) (string, error) {
-	if strings.Count(val, "=") != 1 {
+	if strings.Count(val, "=") < 1 {
 		return "", fmt.Errorf("bad attribute format: %s", val)
 	}
```

Comment:
It was discussed to add a `[a-zA-Z0-9][a-zA-Z0-9_.-]` restriction (https://github.com/docker/docker/pull/9882#issuecomment-70106777) but that didn't make it in the PR (see https://github.com/docker/docker/pull/9882#issuecomment-73987758) apart from being [documented as "should consist of"](https://github.com/docker/docker/pull/9882/files#diff-7dc7275791c2e807567d5e8a89d3f53bR44).

(I would still be in favour of adding that restriction, but that's out of scope here)